src/main/resources/application.yml

```
@@ -3,6 +3,12 @@ spring:
       name: ${artifactId}
       version: ${version}
     jmx.enabled: true
+    data:
+      mypay:
+        jdbc-url: "jdbc:postgresql://\${MYPAY_DB_HOST}:5432/mypay"
+        username: "\${MYPAY_DB_USER}"
+        password: "\${MYPAY_DB_PASSWORD}"
+        driverClassName: "org.postgresql.Driver"
```

let's make configurable also DB_PORT and DB_NAME (put on helm the right value for DB_NAME)
```suggestion
      pu:
        jdbc-url: "jdbc:postgresql://\${PU_DB_HOST}:\${PU_DB_PORT:5432}/\${PU_DB_NAME:dbname}"
        username: "\${PU_DB_USER}"
        password: "\${PU_DB_PASSWORD}"
        driverClassName: "org.postgresql.Driver"
```